src/nvim/po/zh_TW.UTF-8.po

```diff
@@ -6272,7 +6289,7 @@ msgstr "沒有這個 mapping 對應"
 
 #: ../undo.c:2330
 msgid "number changes  when               saved"
-msgstr ""
+msgstr "數字改變當        保存"
```

Comment:
These are column headers
```suggestion
msgstr "  編號    改變  時間        保存"
```

Reply:
this is not correct.

Reply:
Look at the output of `:undolist` in a buffer with modifications.